gazebo_plugins/src/gazebo_ros_vacuum_gripper.cpp

```diff
@@ -110,6 +110,26 @@ void GazeboRosVacuumGripper::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf
     return;
   }
 
+  // Parameters for Vacuum Mechanism
+  // max force: [N]
+  if (_sdf->HasElement("maxForce")) {
+    max_force_ = _sdf->GetElement("maxForce")->Get<double>();
+  } else {
+    max_force_ = 20;
+  }
+  // max distance to apply force: [m]
+  if (_sdf->HasElement("maxDistance")) {
+    max_distance_ = _sdf->GetElement("maxDistance")->Get<double>();
+  } else {
+    max_distance_ = 0.05;
+  }
+  // distance to apply friction: [m]
+  if (_sdf->HasElement("minDistance")) {
```

Comment:
I think `minDistance` is a misleading name for this parameter, because it sounds like anything below this distance will not be affected, which is not true.